Source/Paths/CoordinateParser.cs

```diff
@@ -24,7 +24,7 @@ public enum NumState
         public int Position;
         public bool HasMore;
 
-        public CoordinateParserState(ref ReadOnlySpan<char> chars)
+        public CoordinateParserState(scoped ref ReadOnlySpan<char> chars)
```

Comment:
As far as Im understand, the `scoped` keyword wa introduced in C#11. What about compiling for older versions?

Reply:
It will be a compilation error.
We can also pin language version to C#10 if we want that.

Reply:
At the moment I wouldn't want to restrict the compiler to C# 11, so pinning to C# 10 is indeed an option, at least for a time. If there is an option to make it compile under both versions (using #ifdef) it would be better, but I don't know if there is a possibility to do this.